src/ai/backend/client/cli/run.py

```diff
@@ -319,7 +319,9 @@ def prepare_mount_arg(
               metavar='NAME[=PATH]', type=str, multiple=True,
               help='User-owned virtual folder names to mount. '
                    'If path is not provided, virtual folder will be mounted under /home/work. '
-                   'All virtual folders can only be mounted under /home/work. ')
+                   'Relative paths are under /home/work. '
+                   'If you want different paths, names should be absolute paths. '
+                   'Please use different path from the linux system folders. ')
```

Comment:
```suggestion
                   'When the target path is relative, it is placed under /home/work '
                   'with auto-created parent directories if any. '
                   'Absolute paths are mounted as-is, but it is prohibited to '
                   'override the predefined Linux system directories.')
```

Please update other places as well.